README.md

```diff
@@ -410,6 +412,30 @@ release most of the models ourselves.
 
 </details> 
 
+
+### VLM Support Matrix
+
+For most VLMs, we typically support the default quantization method, which involves quantizing only the language component while excluding the visual component. Besides, we also support quantizing non-text modules of models that follow the Hugging Face standard, i.e., those with a typical processor.
```

Comment:
support the default quantization method,  method->configuration

Reply:
i.e., those with a typical processor, though inference may have some issues  due to model architecture or kernel limitations.